hack/update-generated-protobuf.sh

```diff
@@ -2,8 +2,8 @@
 source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
 if [[ -z "$(which protoc)" || "$(protoc --version)" != "libprotoc 3.0."* ]]; then
-  echo "Generating protobuf requires protoc 3.0.0-beta1 or newer. Please download and"
-  echo "install the platform appropriate Protobuf package for your OS: "
+  echo "Generating protobuf requires protoc >= 3.0.0-beta1, protoc < 3.1. Please"
```

Comment:
Can you elaborate? Are you just wanting me to simplify the wording to something like 3.0.x ? (Assuming that nobody will try to use the 3.0.0-alpha(x) versions)